src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -551,7 +547,7 @@ public static boolean postOcsBatch(String payload) {
   public static boolean postBatch(String payload) {
     boolean requestSuccess =
         RequestHttps(
-            OSIsoftConfig.getTargetURL() + "batch/",
+            OSIsoftConfig.getServerUrl() + "/batch",

```

Comment:
ending "/" removed

Reply:
This was intentional. The API works with or without the trailing slash, but all other requests in the connector do not end with a trailing slash. I can move this to another commit to include a commit message about this